usr/share/rear/output/PXE/default/820_copy_to_net.sh

```diff
@@ -16,7 +16,7 @@ case "$scheme" in
         LogPrint "Transferring PXE files to $OUTPUT_URL"
         for result_file in "${RESULT_FILES[@]}" ; do
             LogPrint "Transferring file: $result_file"
-            lftp -c "open $OUTPUT_URL; mput $result_file" || Error "Problem transferring '$result_file' to $OUTPUT_URL"
+            lftp -c "open $OUTPUT_URL; $LFTP_PARAMS mput $result_file" || Error "Problem transferring '$result_file' to $OUTPUT_URL"
```

Comment:
@vigri 
I would appreciate it if you also could improve our old and rather meaningless
Error message into something that is more helpful for the user, e.g. something like
```
lftp -c "open $OUTPUT_URL; $LFTP_PARAMS mput $result_file" || Error "lftp failed to transfer '$result_file' to $OUTPUT_URL (lftp exit code is $?)"
```
or whatever you like that could help the user to understand what went wrong.